src/ServicePulse.Host/app/js/app.constants.js

```diff
@@ -5,7 +5,7 @@
         .constant('showPendingRetry', false)
         .constant('scConfig', {
             service_control_url: 'http://localhost:33333/api/',
-            monitoring_urls: ['http://localhost:33633/']
+            monitoring_urls: ['http://localhost:1234/']
```

Comment:
SC.Monitoring should change to have it's default be reasonable. 1234 might work for now, but 33633 will be easier for development and debugging later. Once SC.Monitoring has been installed, it is extremely unlikely that 1234 will remain the port, whereas 33633 has a much higher chance.